community/SDK-GOVERNANCE.md

```diff
@@ -10,7 +10,42 @@ The community is organized as follows:
 
 ## New maintainers
 
-TBD
+Maintainers are a crucial part of this community. They keep our projects alive
+and ensure both development and maintenance. We define a set of criteria,
+followed by a vote, to include an active contributor as maintainer of one or
+more SDK projects. An active contributor may propose themselves as a maintainer,
+although we encourage the existing maintainers to nominate active contributors
+proactively.
+
+The criteria for a contributor to become a maintainer for a given SDK repository
+are:
+
+1. At least 20 reviews on eligible PRs
+1. At least 20 submitted eligible PRs as author
+
+When we refer to the term _eligible pull requests_, we explicit mean pull
+requests with substantial contributions to the project. Because a _"substantial
+contribution"_ may vary between the different technologies used, each project
+defines in the `CONTRIBUTING.md` (available at the root of the Github
+Repository) what _eligible PR_ means for it.
+
+If a contributor does not meet these criteria, he should not be considered for
+approval as a maintainer of the project, **unless** the situation defined in
+[handover the project](#handover-to-a-new-maintainergroup-of-maintainers)
+applies.
+
+Once a contributor has met the above criteria, he, or an existing maintainer,
+can ask to proceed with a vote using the
+[Asynchronous voting process](#asynchronous-voting-process).
+
+The voting criteria are:
+
+- 1 week to vote
+- At least 2/3 of the votes cast agree to the proposal
+- _sdk-x maintainers_ are entitled to vote
```

Comment:
It wasn't completely clear to me that "sdk-x" meant the maintainers of the particular SDK. Maybe something descriptive like `_existing maintainers of the particular SDK repository_`?

Reply:
It's explicitly explained at the beginning of the document